### DISCUSSION TITLE
Change data folder location

### DISCUSSION THREAD

**Discussion by beniblanco78** [keycloak/keycloak#14528] [Archive]
Hello,

In the previous versions of keycloak we were able to change the data folder location using the jboss.server.data.dir java option.
It seems that this is not working anymore in the Quarkus distribution.
Is is still possible? and if yes how can we do? (I was not able to find the information in the documentation)

We had the same issue with the log directory (but we found the new option "--log-file" to solve it)

Thanks in advance,
Nicolas

**Comment by beniblanco78** [comment]
Hello,
Any update on this request?
Thanks,
Nicolas

**Reply by darius-m** [reply]
If there is no such configuration option, you could probably create a symbolic link from the Keycloak data directory to the expected location.
